execution-plan/src/api_request.rs

```diff
@@ -80,6 +80,43 @@ impl ApiRequest {
                 log_req(events);
                 session.run_command(cmd_id, cmd).await?
             }
+            Endpoint::ImportFiles => {
+                let mut args_iter = arguments;
+                let arg_opt_import_files_struct = args_iter.next();
+
+                let Some(arg_import_files_struct_prim) = arg_opt_import_files_struct else {
+                    return Err(ExecutionError::General {
+                        reason: "Endpoint::ImportFiles requires an ImportFiles struct".to_string(),
+                    });
+                };
+
+                let arg_import_files_struct =
+                    mem.get_in_memory::<kittycad_modeling_cmds::ImportFiles>(arg_import_files_struct_prim, "", events)?;
```

Comment:
```suggestion
                    mem.get_in_memory::<kittycad_modeling_cmds::ImportFiles>(arg_import_files_struct_prim, "ImportFiles struct", events)?;
```
this will be more helpful in the debugger